lib/workload/stateless/stacks/sequence-run-manager/compose.yml

```diff
@@ -0,0 +1,10 @@
+include:
+  - path:
+#      - ../../../../../shared/mock-ica.yml
+      - ../../../../../shared/mock-aws.yml
+      - ../../../../../shared/mock-db.yml
```

Comment:
"You are now in level 5" _(sophon, 3 body problems)_

Just for laugh, ignore pls. All good.